### DISCUSSION TITLE
error: src refspec master does not match any

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [Andrew-Chen-Wang/github-wiki-action#16] [Q&A] [answered]
Hey there, it's me again, with other noob questions/problems. 😅 
I'm getting this error in the action: `error: src refspec master does not match any`.
How can I solve?

My yml: https://github.com/201flaviosilva-labs/javascript-utils/blob/main/.github/workflows/DeployWiki.yml
Run error: https://github.com/201flaviosilva-labs/javascript-utils/actions/runs/3203594863/jobs/5234042258

**Comment by Andrew-Chen-Wang** [comment, marked as answer]
GitHub must have changed the wiki git branch from master to main. Good catch!

Edit: GitHub didn't. OP's wiki was not enabled

**Reply by 201flaviosilva** [reply]
Hummm, so should I change the branch name to master?

**Reply by Andrew-Chen-Wang** [reply]
It's not something you're able to specify in GitHub actions inputs. It's an entrypoint.sh change where we push to master. If you git clone your wiki repo (you can see how we do it in entrypoint.sh), you'll see the default branch.

**Reply by 201flaviosilva** [reply]
Sorry, I don't think I got it, what I need to do to solve it? 😅

**Reply by Andrew-Chen-Wang** [reply]
sorry i meant i'll push a new release to solve it

**Reply by 201flaviosilva** [reply]
Ahh, ok, ok, thank you very much :)

Please let me know when it's ok :)

**Reply by Andrew-Chen-Wang** [reply]
I don't see a Wiki enabled in your repository. Please enable Wiki and create a temporary Home.md

**Reply by 201flaviosilva** [reply]
Ohhhh, fixed, thanks for everything 😁
